### PR TITLE
refactor: Renamed package

### PR DESCRIPTION
**Pull request by frgfm**
This PR updates package name and updates CI cache.

If I missed something, please let me know pray

### CODE REVIEW

File: pyroengine/__init__.py [hunk 1]
```diff
@@ -5,4 +5,3 @@
 # See the LICENSE file in the root of this repository for complete details.
 
 from .version import __version__
-from pyronearEngine import inference
```
Comment (fe51) [inline pyroengine/__init__.py:8]: You delete the line because it was useless ? Otherwise, we should just change `pyronearEngine ` to `pyroengine` ?

**Review by fe51 — APPROVED** [review]
Thanks for the PR.

exept one question, it seems okay to me !

File: pyroengine/__init__.py (hunk 1 above)
Comment (frgfm) [reply pyroengine/__init__.py:8]: Well, not useless but it meant:
`from pyroengine import *` would import everything declared in ̀inference`

I thought that `from pyronear.inference import ...` is a better & cleaner interface.

**Review by MateoLostanlen — APPROVED** [review]
All good :)

**Pull request merged**